package.json

```diff
@@ -38,20 +38,21 @@
     "karma-webpack": "^3.0.0",
     "minimist": "^1.2.0",
     "mocha": "^5.2.0",
-    "prop-types": "^15.6.1",
-    "react": "^16.3.2",
     "react-dom": "^16.4.1",
     "sinon": "^6.1.3",
     "webpack": "^4.2.0"
```

Comment:
You can update to `4.16`. And the `webpack-cli` is missing.

Reply:
I don't think that webpack-cli is required.